GherkinSyncTool/README.md

```diff
@@ -0,0 +1,90 @@
+# GherkinSyncTool
+
+GherkinSyncTool is an open-source console application, that synchronizes tests scenarios in
+a [gherkin syntax](https://cucumber.io/docs/gherkin/) also known as feature files with a tests management system or any
+other destinations. Currently, a TestRail synchronizer is implemented
+
+## Installation
+
+- Install [.NET](https://dotnet.microsoft.com/download)
+- Build solution
+```
+dotnet build
+```
+- Run app
+```
+dotnet .\.\GherkinSyncTool\bin\Debug\net5.0\GherkinSyncTool.dll
+```
+
+## Configuration
+
+GherkinSyncTool could be configured in three ways. The priority corresponds to the list order.
+
+1. appsettings.json
+2. Environment variables
+3. Command-line arguments
+
+### TestRail
+
+For the correct work of the tool, the TestRail template should have custom fields, which are presented in the table below. The template should not contain the required fields.
+
+|System Name|Type|
+|----|----|
+|`preconds`| Text type|
+|`steps_separated`| Step type|
+|`custom_tags`| String type|
+
+### Common configuration
+
+|Parameter|Description|Required|
+|----|----|----|
+|BaseDirectory|Absolute or relevant path to a folder that contains *.feature files|Yes|
```

Comment:
I guess it should be "Absolute or relative to application folder"

Reply:
Fixed